src/roiextractors/extractors/tiffimagingextractors/scanimagetiff_utils.py

```diff
@@ -120,8 +120,14 @@ def parse_metadata(metadata: dict) -> dict:
     else:
         num_planes = 1
         frames_per_slice = 1
-    active_channels = parse_matlab_vector(metadata["SI.hChannels.channelsActive"])
-    channel_indices = np.array(active_channels) - 1  # Account for MATLAB indexing
+
+    channel_availability_keys = ["SI.hChannels.channelSave", "SI.hChannels.channelsActive"]
```

Comment:
Can you add a note in the docstring of this function?

Reply:
Good idea.

Reply:
Done.